PetstoreAPI/PetStoreRequester.cs

```diff
@@ -18,11 +11,13 @@ class PetStoreRequester
         /// <param name="args"></param>
         static void Main(string[] args)
         {
+            //Source the Website URl from the config file
             IConfiguration config = new ConfigurationBuilder()
                 .AddJsonFile(@"C:\Users\LiamWard\source\repos\PetstoreAPI\PetstoreAPI\appsettings.json")
```

Comment:
We don't want to hardcode the appsettings location - otherwise it won't run on any other machine. See if you can load the file relative to the location of the running executable.